### ISSUE TITLE
Provide an export format flag, or detect piping

### ISSUE THREAD

**Issue by rixx** [thibaudcolas/curlylint#20]
### Is your proposal related to a problem?

Scrollback in terminals is often inconvenient, so I'm in a habit to pipe tool output into a temporary file. Introducing curlylint in a larger project can produce thousands of annotations!

But curlylint output always includes console colours, so the file reads like this:

```
[2m6:4[0m	Bad indentation, expected 8, got 4	indent
[2m7:4[0m	Bad indentation, expected 8, got 4	indent
[2m14:0[0m	Bad indentation, expected 4, got 0	indent
```

### Describe the solution you’d like

Many tools either detect that their output is piped somewhere (which is a bit meh, I think) or provide a flag to force unformatted output. Down the line, different reporting formats might be a thing in any case, so maybe a flag like `--report=plain`, so later we could also have `--report=json`?

**Comment by rixx** [comment]
I'm sorry, I can't read. There's a `--format` flag. Sorry for the noise!

**Comment by thibaudcolas** [comment]
👍  no problem! If you have a suggestion for documenting this better / the need for more "plain formatting" options please let me know. At least I can see how it’d be useful to have a "Guides" section on the website, with a succinct "integration with other tools" or similar.

---

For future reference, the current way to disable all color output would be either of:

```sh
# Plain text
curlylint --quiet --format compact your_templates/
# JSON
curlylint --quiet --format json your_templates/
```
